zds/tutorialv2/models/models_database.py

```diff
@@ -93,17 +93,14 @@ class Meta:
                                       blank=True, null=True, max_length=80, db_index=True)
     sha_draft = models.CharField('Sha1 de la version de rédaction',
                                  blank=True, null=True, max_length=80, db_index=True)
-    beta_topic = models.ForeignKey(Topic,
-                                   verbose_name='Contenu associé',
-                                   default=None,
-                                   null=True)
+    beta_topic = models.ForeignKey(Topic, verbose_name='Sujet beta associé', default=None, blank=True, null=True)
```

Comment:
non car on autorise à avoir null donc c'est bon.

2016-04-01 13:09 GMT+02:00 Laville Augustin <notifications@github.com>:

> In zds/tutorialv2/models/models_database.py
> <https://github.com/zestedesavoir/zds-site/pull/3475#discussion_r58191546>
> :
>
> > @@ -93,17 +93,14 @@ class Meta:
> >                                        blank=True, null=True, max_length=80, db_index=True)
> >      sha_draft = models.CharField('Sha1 de la version de rédaction',
> >                                   blank=True, null=True, max_length=80, db_index=True)
> > -    beta_topic = models.ForeignKey(Topic,
> > -                                   verbose_name='Contenu associé',
> > -                                   default=None,
> > -                                   null=True)
> > +    beta_topic = models.ForeignKey(Topic, verbose_name='Sujet beta associé', default=None, blank=True, null=True)
>
> C'est pour tous les attributs. Là tu obliges ton objet à avoir un
> beta_topic associé.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/zestedesavoir/zds-site/pull/3475/files/e36f8799c2b713ca93b8a0de5dcef87818df4308#r58191546>
>